crates/bevy_render/macros/src/as_bind_group.rs

```diff
@@ -127,7 +119,7 @@ pub fn derive_as_bind_group(input: TokenStream) -> TokenStream {
             let attr_ident = if let Some(ident) = attr.path.get_ident() {
                 ident
             } else {
-                continue;
+                return Err(Error::new_spanned(attr, "Expected identifier"));
```

Comment:
This will include "non-AsBindGroup attrs". Dropping an eprintln on `ident` prints `doc` for the bevy_pbr crate (in addition to things like `texture`). I think it is better to use `continue` here so we don't try to impose restrictions on other macros' data.